app/helpers/link_to_helper.rb

```diff
@@ -254,6 +254,20 @@ def unhappy_url(info_request = nil)
     end
   end
 
+  # Google Analytics
+
+  # if used, value must an integer
+  def analytics_event_code(category, action, label=nil, value=nil, label_is_script=false)
+    value = 1 unless value
```

Comment:
note to self: proposed new method signature:

`def analytics_event_code(event_category, event_action, options={})`

(original method name used for readability purposes)